src/db-service.ts

```diff
@@ -203,20 +232,18 @@ export class ItemValidationService {
 
   /**
    * Update an entry for the automatic validation process in DB
-   * @param itemValidationEntry entry with updated data
+   * @param status new status of process, failure or success
```

Comment:
I think you can follow jsdoc https://jsdoc.app/tags-param.html
You can specify `string` as the type.